ddtrace/contrib/kombu/__init__.py

```diff
@@ -46,8 +48,14 @@
             _w.simplefilter("ignore", DeprecationWarning)
             from . import patch as _  # noqa: F401, I001
 
-        # Expose public methods
-        from ddtrace.contrib.internal.kombu.patch import get_version
-        from ddtrace.contrib.internal.kombu.patch import patch
+        from ddtrace.contrib.internal.kombu.patch import get_version  # noqa: F401
+        from ddtrace.contrib.internal.kombu.patch import patch  # noqa: F401
 
-        __all__ = ["patch", "get_version"]
+
+deprecate(
+    ("%s is deprecated" % (__name__)),
+    message="Avoid using this package directly. "
+    "Use ``ddtrace.auto`` or the ``ddtrace-run`` command to enable and configure this integration.",
```

Comment:
This won't work because `kombu` doesn't have auto-instrumentation enabled by default. Until we change that, we should remove this line for `kombu` and the other integrations that don't have auto-instrumentation enabled by defalut

Reply:
Can we go with this instead?

```suggestion
    "Use ``ddtrace.patch(kombu=True)`` command to enable and configure this integration.",
```